src/components/collection/Summary.js

```diff
@@ -91,6 +130,39 @@ function Summary({
     setOpen(false);
   };
 
+  const handleFavorite = () => {
+    updateFavoriteItem({
+      memberId: member.get('id'),
+      extra: member?.get('extra'),
+      itemId,
+      action: 'add',
```

Comment:
I would have created two mutations instead. Add, and remove, instead of depending on some string.